share/completions/magento.fish

```diff
@@ -0,0 +1,720 @@
+###
+# executes fish MySQL commands
+#
+# To make this work, you'll need to create a __fish user in your
+# database, using the following mysql commands:
+#
+#   mysql> CREATE USER '__fish'@'localhost' IDENTIFIED BY 'your random pw';
+#   mysql> GRANT SELECT, SHOW ON * . * TO '__fish'@'localhost';
+#   mysql> FLUSH PRIVILEGES;
+#
+# If no credentials are given, it will simply print nothing, so further commands
+# won't show suggestions
+###
+function __fish_print_magento_mysql_command -d "Executes a MySQL command using the fish user"
+
+### IMPORTANT: ENTER FISH CREDENTIALS HERE #####
+  set -l fish_mysql_user __fish
+  set -l fish_mysql_pass fish_12479284903518
+################################################
+
+  set -l result (mysql -u{$fish_mysql_user} -p{$fish_mysql_pass} -e $argv --skip-column-names 2>&1 | grep -v "mysql:")
+  if [ (echo $result | grep ERROR) ]
+    return 1
+  else
+    for i in $result
+      echo $i
+    end
+    return 0
+  end
+end
+
+function __fish_print_magento_url_protocols -d "Shows all available URL protocols"
+  echo http://\t"HTTP"
+  echo https://\t"HTTPS"
```

Comment:
In general, if suggestions just duplicate information you can already see we prefer leaving them. So this would just echo "http://" and "https://".